core/storage/statistics/gae_models.py

```diff
@@ -970,19 +1001,20 @@ class PlaythroughModel(base_models.BaseModel):
     {{exp_id}}.{{random_hash_of_16_chars}}
     """
     # ID of the exploration.
-    exp_id = ndb.StringProperty(indexed=True)
+    exp_id = ndb.StringProperty(indexed=True, required=True)
     # Version of the exploration.
-    exp_version = ndb.IntegerProperty(indexed=True)
+    exp_version = ndb.IntegerProperty(indexed=True, required=True)
     # Type of the issue.
-    issue_type = ndb.StringProperty(indexed=True)
+    issue_type = ndb.StringProperty(
+        indexed=True, required=True, choices=ALLOWED_ISSUE_TYPES)
     # The customization args dict for the given issue_type.
-    issue_customization_args = ndb.JsonProperty(default=None)
+    issue_customization_args = ndb.JsonProperty(default=None, required=True)
```

Comment:
Why do you need a default value at all here? None is never valid, right?

Reply:
Done.